### ISSUE TITLE
Typo in makeoxt

### ISSUE THREAD

**Issue by n8marti** [silnrsi/oxttools#15]
In looking to understand how makeoxt works I came across a typo in the zipnfcfile function in line 67 of makeoxt:
```python
        with oepn(affix) as fd:
```
surely that should be:
```python
        with open(affix) as fd:
```

**Comment by DavidLRowe** [comment]
You are correct, @n8marti ! I guess no one has successfully used that feature.

**Comment by DavidLRowe** [comment]
fixed in release v0.4 (https://github.com/silnrsi/oxttools/releases/tag/v0.4)
